compiler/p/codegen/OMRRegisterDependency.cpp

```diff
@@ -844,74 +844,86 @@ void TR_PPCRegisterDependencyGroup::assignRegisters(TR::Instruction   *currentIn
       {
       map.addDependency(_dependencies[i], i);
 
-      virtReg = _dependencies[i].getRegister();
       dependentRegNum = _dependencies[i].getRealRegister();
 
       if (dependentRegNum != TR::RealRegister::SpilledReg)
          {
-         if (virtReg->getKind() == TR_GPR)
-            num_gprs++;
-         else if (virtReg->getKind() == TR_FPR)
-            num_fprs++;
-         else if (virtReg->getKind() == TR_VRF)
-            num_vrfs++;
+         virtReg = _dependencies[i].getRegister();
+         switch (virtReg->getKind())
+            {
+            case TR_GPR:
+               ++numGPRs;
+               break;
+            case TR_FPR:
+               ++numFPRs;
+               break;
+            case TR_VRF:
+               ++numVRFs;
+               break;
+            case TR_CCR:
+               if (!virtReg->getAssignedRegister())
```

Comment:
`getAssignedRegister() != NULL && getBackingStorage() != NULL` is effectively what we use on z Systems. This should work fine. Thanks for the followup.